database/factories/CommentFactory.php

```diff
@@ -20,7 +20,7 @@
         'playlist_id' => function () {
             return factory(App\Models\Playlist::class)->create()->id;
         },
-        'collection_id' => $faker->randomNumber(),
+        //'collection_id' => $faker->randomNumber(),
```

Comment:
Can we eliminate this? It seems like this column may have been deleted since the Shift stubs were generated.

Reply:
yes